devel/libcoverart/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        metabrainz libcoverart 1.0.0 release-
+github.tarball_from releases
+categories          devel
+platforms           darwin
+maintainers         nomaintainer
+license             LGPL-2.1+
+
+description         C/C++ library for accessing the MusicBrainz Cover Art Archive
+
+long_description    libcoverart is a C/C++ library for accessing the MusicBrainz Cover Art Archive
```

Comment:
In order not to repeat yourself, you can reference existing variables. For example I often write:

```
long_description    ${name} is a {*}${description}.
```